app/models/article_price.rb

```diff
@@ -21,11 +21,11 @@ class ArticlePrice < ApplicationRecord
   #   @return [Array<OrderArticle>] Order articles this price is associated with.
   has_many :order_articles
 
-  localize_input_of :price, :tax, :deposit
+  localize_input_of :price, :tax, :deposit, :supplier_price
 
   validates_presence_of :price, :tax, :deposit, :unit_quantity
   validates_numericality_of :price, :greater_than_or_equal_to => 0
   validates_numericality_of :unit_quantity, :greater_than => 0
-  validates_numericality_of :deposit, :tax
+  validates_numericality_of :deposit, :tax, :supplier_price
```

Comment:
Would the supplier price also need a check for >= 0?

Reply:
done